### ISSUE TITLE
Blocked - v9 newstonsoft.dll dependency

### ISSUE THREAD

**Issue by CharlesFichter** [launchdarkly/.net-client#73]
We are seeing an issue with the latest v9 of newtonsoft.dll from the launchdarkly NuGet - (might be 9.1) -

We have updated our binding redirect to catch v6 through v10 of newtonsoft.dll (we have several other newtonsoft.dll version dependencies --)

But launch darkly is failing stating it wants specifically only v9 when v10 is present in bin.

Our customer does not want us to do the multiple redirects (different dlls in different folders, 2 different binding redirects).  

What is the suggested solution/work around?  Would v11 newtonsoft.dll potentially solve this? This would create an issue with the customer, however, they would then have to test legacy dependencies on v10.

Thanks

**Comment by eli-darkly** [comment]
Thanks for bringing this to our attention. We're looking into the best way to handle it.

**Comment by eli-darkly** [comment]
Also, could you let us know exactly what your binding redirect for newtonsoft.dll looks like at present?

**Comment by CharlesFichter** [comment]
Hi Eli –
We did more troubleshooting on this.
It was extremely odd, I must say – but the customer’s environment we are working in is super fragile and inconsistent across the environment.
I was able to get launchdarkly working fine with v10 of the newtonsoft.dll in my environment with a binding redirect – the developer machine we were testing from had other issues related to group policy and network access – but the exceptions being thrown from Sitecore was throwing us off, but I think we put this to rest. I’ve been on vacation this week, and I’ve been talking with the team, I think we are g2g.

Thanks for checking in though.
